doctr/utils/multithreading.py

```diff
@@ -34,5 +34,7 @@ def multithread_exec(func: Callable[[Any], Any], seq: Iterable[Any], threads: Op
     # Multi-threading
     else:
         with ThreadPool(threads) as tp:
-            results = tp.map(func, seq)  # type: ignore[assignment]
+            # ThreadPool's map function returns a list, but seq could be of a different type
+            # That's why wrapping result in map to return iterator
+            results = map(lambda x: x, tp.map(func, seq))  # type: ignore[assignment]
```

Comment:
Mmmmh, could we have a latency check here please? :pray: 

I just want to make sure we don't increase latency too much considering how critical this function is for data loading :sweat_smile:

Reply:
I have done two tests: one on png image and one on 15 pages pdf. I have Ryzen 5800h with 8 cores and 16 threads. Results are approximately the same
![image](https://user-images.githubusercontent.com/67135906/186606880-75f2f58a-f697-412b-a2d2-32d7b96e5178.png)
![image](https://user-images.githubusercontent.com/67135906/186607205-fb9da5c3-cc9c-414b-a646-448b24b02ae2.png)

Reply:
Hi @mtvch thanks a lot looks good to me :+1: